src/Avalonia.Controls/Templates/TemplateExtensions.cs

```diff
@@ -1,17 +1,41 @@
 // Copyright (c) The Avalonia Project. All rights reserved.
 // Licensed under the MIT license. See licence.md file in the project root for full license information.
 
-using System;
 using System.Collections.Generic;
 using System.Linq;
-using Avalonia.Controls;
 using Avalonia.Styling;
 using Avalonia.VisualTree;
 
 namespace Avalonia.Controls.Templates
 {
     public static class TemplateExtensions
     {
+        /// <summary>
+        /// Gets a named control from a templated control's template children.
+        /// </summary>
+        /// <param name="template">The control template.</param>
+        /// <param name="name">The name of the control.</param>
+        /// <param name="templatedParent">The templated parent control.</param>
+        /// <returns>An <see cref="IControl"/> or null if the control was not found.</returns>
+        public static IControl FindName(this IControlTemplate template, string name, IControl templatedParent)
+        {
```

Comment:
Add null checks for `template` and `templatedParent`?

Reply:
Added null checks, thanks!